README.md

```diff
@@ -60,15 +60,14 @@ We use http://www.apache.org/licenses/LICENSE-2.0
 ### Spring Integration
 
 As the spring projects
-[removed the embed mongo support in 2.7.0](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes#springmongodbembeddedfeatures-configuration-property-removed)
+[removed the embed mongo support in 3.0.0](https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#embedded-mongodb)
 you should consider to use one of these integration projects.
 It should behave mostly like the original spring integration, but there are some minor differences:
 - version in 'spring.mongodb.embedded.version' is used in package resolver and is not matched against version enum.
 - 'spring.mongodb.embedded.features' is not supported (not the way to change the config of mongodb)
 
 If you have any trouble in using them fell free to create an issue.
 
-- [Spring 2.5.x](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo.spring/tree/spring-2.5.x)
```

Comment:
Spring 2.5.x and 2.6.x support is still there. So i would not change that.

Reply:
@michaelmosmann I didn't count its commercial support, but 2.5.x also has been EOLed even for commercial support. I'll reinstate the entry for 2.6.x, but please let me know if you still want 2.5.x to be reinstated, too.

See https://spring.io/projects/spring-boot#support

Reply:
@izeye I do not provide commercial support for spring 2.5.x but there are still users out there using it. So i would leave it:)